src/App.js

```diff
@@ -87,17 +96,26 @@ const App = () => {
     };
     
     return (
-        <>
-            <h1>Todo List</h1>
-            <hr/>
-            <AddTodoForm onAddTodo={addTodo}/>
-            {isLoading ? (
-                <p>Loading...</p>
-            ) : (
-                <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>
-            )}
-            <hr/>
-        </>
+        <BrowserRouter>
+            <Routes>
+                <Route path={"/"} element={
```

Comment:
Nitpick:
For the `path` property, you only need braces around expressions (something that needs to be calculated). For non-interpolated strings (`${value}`), you can just assign `path` as a string without braces: `path="/"`.